irc/tests.py

```diff
@@ -80,4 +92,65 @@ def test_msg_types_changed(self):
         """
         Tests that no message types have changed
         """
-        self.assertTrue(MSG_TYPES == MSG_TYPES_STANDARD)
\ No newline at end of file
+        self.assertTrue(MSG_TYPES == MSG_TYPES_STANDARD)
+
+    def test_parse_method(self):
```

Comment:
Lepo napisan test, ki pokrije dokaj kompleksno funkcijo.

Edini problem je, da test pokrije code coverage, ampak ne preveri pa funkcionalnosti. E.g. ce odstranis vso kodo iz parse metode in vrnes samo 'OK' bo test se vedno passal 😂

Kul bi bilo dodati se par dodatnih assertov, na primer:

1. Ce je ERROR msg ali se msg_type nastavi na 'E'
2. Ce se msg_type nastavi na 'M' ali 'PM' pravilno.
3. Ce se message nastavlja pravilno.
4. Ce je link v messagu, ali se Link() object ustvari.
etc

Vecino teh chackov lahko naredis tako, da po klicu pogledas v bazo in probas najt zadnji Irc()/Link() objekt...